engine/src/main/java/org/hibernate/search/bridge/util/impl/NumericFieldUtils.java

```diff
@@ -24,6 +41,9 @@
 
 	private static final Log log = LoggerFactory.make();
 
+	private static final Class<?>[] NUMERIC_JAVA_TIME_TYPES = { Year.class, Instant.class, LocalDate.class, LocalDateTime.class, YearMonth.class,
```

Comment:
I didn't have error when building with JDK7.
Isn't static only initialized when you call it? This shouldn't get called if you are using JDK7